caching/src/test/scala/org/allenai/common/cache/JsonQueryCacheSpec.scala

```diff
@@ -1,27 +1,24 @@
 package org.allenai.common.cache
 
-import java.io.File
-import org.allenai.common.{ GitVersion, Version, cache }
 import org.allenai.common.testkit.UnitSpec
-import org.scalatest.BeforeAndAfterAll
-import spray.json.DefaultJsonProtocol
-import sys.process._
+
+import com.fiftyonred.mock_jedis.MockJedisPool
```

Comment:
Cool! I don't the original author found this library. We had something in place originally that was buggy (either the included mock library, or something else . . .)